tests/queries/0_stateless/02735_parquet_encoder.sql

```diff
@@ -0,0 +1,166 @@
+set output_format_parquet_use_custom_encoder = 1;
```

Comment:
```suggestion
-- Tags: no-fasttest

set output_format_parquet_use_custom_encoder = 1;
```

We don't link parquet lib in fasttest